File: async-scala-client/src/main/scala/io/swagger/client/model/PayRecCalculatedTaxSummaryForService.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class PayRecCalculatedTaxSummaryForService (
  numberOfLines: Option[Integer],  // Count of lines
subtotal: Option[Double],  // Sum of grossvalues
totalTax: Option[Double],  // Sum of all withholding values
grandTotal: Option[Double],  // Sum all NetValues
pccWithholdingModes: Option[List[PccWithholdingMode]],
taxByType: Option[PayRecCalculatedTaxSummaryForService_taxByType]
)

File: javascript-closure-angular-client/API/Client/PayRecCalculatedTaxSummaryForService.js
goog.provide('API.Client.PayRecCalculatedTaxSummaryForService');

/**
 * @record
 */
API.Client.PayRecCalculatedTaxSummaryForService = function() {}

/**
 * Count of lines
 * @type {!number}
 * @export
 */
API.Client.PayRecCalculatedTaxSummaryForService.prototype.numberOfLines;

/**
 * Sum of grossvalues
 * @type {!number}
 * @export
 */
API.Client.PayRecCalculatedTaxSummaryForService.prototype.subtotal;

/**
 * Sum of all withholding values
 * @type {!number}
 * @export
 */
API.Client.PayRecCalculatedTaxSummaryForService.prototype.totalTax;

/**
 * Sum all NetValues
 * @type {!number}
 * @export
 */
API.Client.PayRecCalculatedTaxSummaryForService.prototype.grandTotal;

/**
 * @type {!Array<!API.Client.PccWithholdingMode>}
 * @export
 */
API.Client.PayRecCalculatedTaxSummaryForService.prototype.pccWithholdingModes;

/**
 * @type {!API.Client.PayRecCalculatedTaxSummaryForService_taxByType}
 * @export
 */
API.Client.PayRecCalculatedTaxSummaryForService.prototype.taxByType;

